src/hist/core.py

```diff
@@ -241,3 +241,24 @@ def pull_plot(
         pull_ax.set_ylabel("Pull")
 
         return fig, ax, pull_ax
+
+    def __getitem__(self, index):
+        """
+            Get histogram item.
+        """
+
+        if not hasattr(index, "__iter__"):
+            index = () + (index,)
+
+        t: tuple = ()
+        for i, idx in enumerate(index):
+            if isinstance(idx, complex):
```

Comment:
Then just do:
```python
    t += (bh.loc(idx.imag, idx.real),)
```

Or, you could do this in one go:

```python
t = tuple((bh.loc(idx.imag, idx.real) if isinstance(idx, complex) else idx) for idx in index)
```

Reply:
You'll also want to add a check to see if it's a string, and then wrap in `loc` in that case too.